examples/webxr_vr_handinput.html

```diff
@@ -85,7 +85,11 @@
 
 				container.appendChild( renderer.domElement );
 
-				document.body.appendChild( VRButton.createButton( renderer ) );
+				const sessionInit = {
+					optionalFeatures: [ 'hand-tracking' ]
```

Comment:
Hmm, yeah... I guess in the `handinput` examples it should be `requiredFeatures`?